spec/core/ics-002-client-semantics/README.md

```diff
@@ -253,6 +253,24 @@ A `ClientMessage` is an opaque data structure defined by a client type which pro
 type ClientMessage = bytes
 ```
 
+### Store paths
+
+Client state paths are stored under a unique client identifier.
+
+```typescript
+function clientStatePath(id: Identifier): Path {
+  return "clients/{id}/clientState"
+}
+```
+
+Consensus state paths are stored under a unique combination of client identifier and height:
+
+```typescript
+function consensusStatePath(id: Identifier, height: Height): Path {
+  return "clients/{id}/consensusStates/{height}"
+}
+```
```

Comment:
These two functions are used in ICS 03 in `verifyClientState` and `verifyClientConsensusState`.